serverless.yml

```diff
@@ -49,6 +49,7 @@ resources:
           WriteCapacityUnits: 5
 
 plugins:
+  - serverless-dotenv-plugin
```

Comment:
What's this for? Does it just allow process.env to be referenced?

Reply:
It reads `.env` and injects all its values into the environment of the Lambda when running with sls offline. You can also set `NODE_ENV` and it will pick up `.env.<env>` as I've done with `.env.local` in this PR.